cli.js

```diff
@@ -48,6 +50,7 @@ Promise
 
 		return options;
 	})
+	.then(options => Object.assign({}, options, {noYarn: noYarn(options)}))
```

Comment:
This check should be done in index.js, not here.

And use https://github.com/sindresorhus/has-yarn